src/services/Utils.js

```diff
@@ -64,7 +64,16 @@ function canUseDom() {
   return (typeof window !== 'undefined' && typeof document !== 'undefined' && document.documentElement);
 }
 
+function toString(v) {
+  try {
+    return v.toString();
+  } catch (er) {
+    return '';
```

Comment:
Not obvious behavior. Why not to throw error if there is no `toString` method? In which cases it is so?

Reply:
null & undefined